examples/CMakeLists.txt

```diff
@@ -1,9 +1,18 @@
-cmake_minimum_required(VERSION 3.9.5)
+cmake_minimum_required(VERSION 3.10.2)
 project(ITKMontageExamples)
 
-find_package(ITK REQUIRED
-  COMPONENTS ITKImageIO ITKTransformIO Montage
-  )
+if(NOT ITK_SOURCE_DIR)
+  find_package(ITK REQUIRED COMPONENTS ITKImageIO ITKTransformIO Montage)
+else()
+  # when being built as part of ITK, ITKImageIO and ITKTransformIO
+  # lists of modules are not yet ready, causing a configure error
+  find_package(ITK REQUIRED COMPONENTS
+    Montage
+    ITKTestKernel # includes many I/O modules
```

Comment:
`ITKImageIO` is preferred

Reply:
`ITKImageIO` does not work properly for remote examples when building them as part of ITK. I talked to Francois, and he says that is still WIP 😄

Reply:
There is explanation for it just a few lines above.